Epam.TestAutomation.Web/PageObjects/Pages/SearchPage.cs

```diff
@@ -0,0 +1,29 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Core.Browser;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Web.PageObjects.Pages;
+
+public class SearchPage : BasePage
+{
+    public override string Url => "https://www.epam.com/";
+
+    //first test variables
+    public static string keyWord = "Automation";
+    public string searchUrl = "https://www.epam.com/search?q=Automation";
+    public By searchButton = By.XPath("//span[@class='search-icon dark-iconheader-search__search-icon']");
+    public By searchForm = (By.XPath("//input[@id='new_form_search']"));
+    public By findButton = (By.XPath("//span[@class='bth-text-layer']"));
+
+    public static IEnumerable<IWebElement> listOfArticles =
+        BrowserFactory.Browser.FindElements(By.XPath("//article")).ToList().Take(5);
```

Comment:
Don't use hardcode. What is 5 in this filed?

Reply:
6.	Check that for the first 5 articles entered text is present in any place - header, body, etc.